demos/python_demos/audio_classification_demo/README.md

```diff
@@ -0,0 +1,60 @@
+# Audio Classification Python\* Demo
+
+Demo application for audio classification algorithm.
+
+## How It Works
+
+Upon the start-up the demo application reads command line parameters and loads a network to Inference engine. It uses only audiofiles in `wav` format. Audio could be converted to different samplerate using `-sr/--samplerate` option, if necessary for the model. After reading the audio, it is sliced into clips to fit model input (clips are allowed to overlap with `-ol/--overlap` option) and each clip is processed separately with its own prediction. 
```

Comment:
Suggest that the sample rate expected by the model, ie., 16KHz is stated here. And, state that if the wav file has any other sample rate, the `-sr` option could be used to convert to 16KHz.